galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -294,18 +294,4 @@ def move_content(self, request, *args, **kwargs):
 
         response_data['copy_task_id'] = response_data['remove_task_id'] = move_task.pk
 
-        if settings.GALAXY_DEPLOYMENT_MODE == DeploymentMode.INSIGHTS.value:
-            golden_repo = AnsibleDistribution.objects.get(
-                base_path=settings.GALAXY_API_DEFAULT_DISTRIBUTION_BASE_PATH
-            ).repository
-
-            if dest_repo == golden_repo or src_repo == golden_repo:
-                curate_task = dispatch(
```

Comment:
I see this PR is attached to AAH-1556. AAH-1540 includes stopping the call to the curate task. AAH-1556 rips out out all synclist repos & curate code. We will deploy AAH-1540 first before AAH-1556.